packages/form-js-viewer/src/features/expression-language/FeelersTemplating.js

```diff
@@ -1,11 +1,47 @@
-import { evaluate as evaluateFeelers } from 'feelers';
+import { parser as feelersParser, buildSimpleTree, evaluate as evaluateFeelers } from 'feelers';
 import { isString } from 'min-dash';
+import { getFlavouredFeelVariableNames } from './variableExtractionHelpers';
 
 export default class FeelersTemplating {
 
   constructor() { }
 
-  isTemplate(value) { return isString(value) && (value.startsWith('=') || /{{/.test(value)); }
+  /**
+   * Determines if the given value is a feelers template.
+   *
+   * @param {any} value
+   * @returns {boolean}
+   *
+   */
+  isTemplate(value) { return isString(value) && (value.startsWith('=') || /{{.*?}}/.test(value)); }
```

Comment:
Is it also a template, when it's an expression only?

Reply:
So renaming wouldn't be the right call here because we're basically implementing an interface of the templating module. 

The reason why I am matching for feel is that my template language also handles the same syntax as the expression language. I could probably remove it, but at the same time the templating supports it, so uhm, technically this is the correct implementation. 

Usually the way the code is run is:

if (isExpression): do expression stuff
else if (isTemplate): do template stuff
else: do static stuff

Our API shouldn't care about the specifics of the module implementation, the way it's implemented right now, we are flexible. If I were to make modifications to remove this feel backwards compatibility from feelers, nothing would break in the app. It's a little confusing but at the same time I don't like purposely nerfing feelers to make it less redundant with the feel expression language implementation.

Reply:
Thanks for explaining, makes sense as is 👍